src/util/helpers/truncateSignature.ts

```diff
@@ -0,0 +1,26 @@
+import Web3 from 'web3';
+
+export const isSignatureValid = (signature: string) =>
```

Comment:
We could probably just import `import {isHex} from 'web3-utils'` and use it directly in the `truncateSignature`.

Reply:
good spot! done